docs/release_notes/ibexa_dxp_v4.4.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Ibexa DXP v4.4 adds the improvements to the Welcome Page, PIM and Fastly.
+---
+
+# Ibexa DXP v4.4
+
+**Version number**: v4.4
+
+**Release date**: January 27, 2023
+
+**Release type**: [Fast Track](https://support.ibexa.co/Public/service-life)
+
+## Notable changes
+
+### New welcome page
+
+You will be welcomed to the Ibexa Digital Experience Platform by a newly designed welcome page.
+
+![New Welcome Page](4.4_welcome_page.png)
+
+### All-new Ibexa Commerce packages [[% include 'snippets/commerce_badge.md' %]]
+
+This release deprecates all Commerce packages that you've known from previous releases 
+and brings a completely redesigned and reconstructed Commerce offering.
+
+As part of this effort, two all-new components have been created: Cart and Checkout, 
+that you can use to build your own e-commerce presence. 
+Another addition is the Storefront component that serves as a proof-of-concept 
+and a basis for your customization and extension.
+
+![The new cart view](img/4.4_new_cart.png "The new cart view")
+
+For more information, see [Commerce](https://doc.ibexa.co/en/4.4/commerce/commerce/).
+
+### Fastly Image Optimizer (Fastly IO)
+
+You can now use Fastly IO to serve optimized versions of your images in real time and cache them.
+Fastly can perform multiple transformations on your image,
+for example, cropping, resizing and trimming before serving it to end user.
+Fastly is an external service that requires a separate subscription,
+to learn more see, [Fastly Image Optimizer website](https://docs.fastly.com/en/guides/about-fastly-image-optimizer).
+
+If you already have Fastly IO subscription, you can move to Fastly IO configuration in Ibexa DXP.
```

Comment:
Should we link to the configuration of this module in Ibexa DXP?

Reply:
Yes we will, it was just merged to master, so links will be added before final release.